sentry-log/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "sentry-log"
+version = "0.18.0"
+authors = ["Sentry <hello@sentry.io>"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/getsentry/sentry-rust"
+homepage = "https://github.com/getsentry/sentry-rust"
+documentation = "https://getsentry.github.io/sentry-rust"
+description = """
+Sentry Integration for log and env_logger crates
+"""
+edition = "2018"
+
+[dependencies]
+sentry-core = { version = "0.18.0", path = "../sentry-core", features = ["with_backtrace"] }
+log = { version = "0.4.8", features = ["std"] }
+env_logger = { version = "0.7.1", optional = true }
```

Comment:
Should we make `env_logger` a separate thing? Otherwise, we'd have one log integration again with all different loggers as features.

Reply:
hm… `log` and `env_logger` are pretty tight AFAICS.